app/models/arclight/parents.rb

```diff
@@ -9,7 +9,7 @@ class Parents
     def initialize(ids:, labels:, eadid:)
       @ids = ids
       @labels = labels
-      @eadid = eadid
+      @eadid = eadid.tr('.', '-')
```

Comment:
👍 Yeah, I was thinking the same thing when I was writing this.  Glad you thought that too, I'll add something.